.github/workflows/newclient.yaml

```diff
@@ -0,0 +1,60 @@
+name: New Client
+on:
+  schedule:
+    - cron: '0 2 * * *' # nightly at 2 am UTC
+  push:
+    branches: [ main ]
+    paths:
+      - generation/**
+  pull_request:
+    branches: [ main ]
+    paths:
+      - generation/**
+
+  workflow_dispatch:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pip' # caching pip dependencies
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d' --utc)"
+      - uses: actions/setup-java@v3
+        with:
+          distribution: zulu
+          java-version: 11
+      - run: java -version
+      - uses: actions/cache@v3
+        id: mvn-cache
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-unified-${{ steps.date.outputs.date }}
+      - name: Install dependencies
+        run: pip install -r generation/new_client/requirements.txt
+      - name: Generate
```

Comment:
Can you add source code comment (or name) "regenerating existing apikeys module"?

Reply:
Yep